src/parser/tests.rs

```diff
@@ -130,8 +131,8 @@ fn functions() {
 
 #[test]
 fn lets() {
-    assert!(parse("let x1 = x2 in x3").is_some());
-    assert!(parse("x (let x1 = x2 in x3) y").is_some());
+    assert!(parse("let x1 = x2 in x3").is_ok());
```

Comment:
Mostly nitpicking, but I guess this won't print anything helpful if the parsing fails. Maybe you could have something more verbose with

```suggestion
    assert_matches!(parse("let x1 = x2 in x3"), Ok(..));
```